src/overview/main.jsx

```diff
@@ -8,13 +8,12 @@ import overview from '../overview'
 import './base.css'
 
 // Include development tools if we are not building for production
-let ReduxDevTools = undefined
-if (process.env.NODE_ENV !== 'production') {
-    ReduxDevTools = require('../dev/redux-devtools').default
-}
+export const ReduxDevTools = process.env.NODE_ENV !== 'production' ?
+    require('../dev/redux-devtools').default :
+    undefined
 
 // Set up the Redux store
-const store = configureStore({ReduxDevTools})
+export const store = configureStore({ReduxDevTools})
```

Comment:
We should export `store` and `ReduxDevTools ` to make them reusable on other pages until they are not moved to separate files